packages/cdktf/lib/util.ts

```diff
@@ -90,7 +90,7 @@ export function keysToSnakeCase(object: any): any {
   }
   const keys = Object.keys(object);
   return keys.reduce((newObject: any, key: string) => {
-    if (key === "tags") {
+    if (key === "tags" || key === "environment") {
```

Comment:
Arguably it might be nice to continue applying snake casing to environment variables, since they're typically UPPER_SNAKE_CASE. Although I agree case should be preserved.

Could definitely see the argument, though, that environment variables should simply be left literally as specified (like tags).